### PR TITLE
[Chat] Updated Docs for Inline Image and File Sharing

### PR DESCRIPTION
**Pull request by jpeng-ms**
# What
Updated stories for inline image and file sharing

# Why

Refactor based on customer feedback

# How Tested

Storybook URL https://60c7ae6891f0e90039d7cd54-xerpxgbztu.chromatic.com/

# Process & policy checklist

- [X] I have updated the project documentation to reflect my changes if necessary.
- [X] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx [hunk 1]
```diff
@@ -139,12 +145,27 @@ export const Docs: () => JSX.Element = () => {
...
+          Chat](https://learn.microsoft.com/en-us/azure/communication-services/tutorials/inline-image-tutorial-interop-chat)
+          for details.
+        </Description>
+      </div>
       <div ref={refFileSharing}>
         <Heading>Adding file sharing</Heading>
       </div>
       <div ref={refFileSharingACS}>
-        <Subheading>In Azure Communication Services Chat Thread</Subheading>
+        <Subheading>File Sharing In Azure Communication Services Chat Thread</Subheading>
```
Comment (emlynmac) [inline packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx:168]: The original text seems better; no need to repeat the file sharing with the header right above?

Comment (jpeng-ms) [reply packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx:168]: i agree, but unfortunately, this subheading would be identical anchor tag as inline image causing the anchor not working properly. for example: #In-Azure Communication-Services-Chat-Thread would navigate to inline image section now and will not navigate to file sharing

**Review by vhuseinova-msft — APPROVED** [review]

File: packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx (hunk 1 above)
Comment (emlynmac) [reply packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx:168]: Let's revisit for v7

File: packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx [hunk 2]
```diff
@@ -45,9 +47,13 @@ export const Docs: () => JSX.Element = () => {
...
       scrollToRef(refFileSharing);
-    } else if (url.includes('in-teams-interop-meeting-chat-thread') && refFileSharingTeams.current) {
+    } else if (url.includes('adding-image-sharing') && refImageSharing.current) {
+      scrollToRef(refImageSharing);
+    } else if (url.includes('inline-image-in-teams-interop-meeting-chat-thread') && refImageSharingTeams.current) {
+      scrollToRef(refImageSharingTeams);
+    } else if (url.includes('file-sharing-in-teams-interop-meeting-chat-thread') && refFileSharingTeams.current) {
       scrollToRef(refFileSharingTeams);
-    } else if (url.includes('in-azure-communication-services-chat-thread') && refFileSharingACS.current) {
+    } else if (url.includes('file-sharing-in-azure-communication-services-chat-thread') && refFileSharingACS.current) {
```
Comment (palatter) [inline packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx:56]: Does this mean the original URL is no longer valid?

Comment (jpeng-ms) [reply packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx:56]: yes, but MSDoc has been updated few weeks ago and no longer pointing to this tag

**Review by emlynmac — APPROVED** [review]

**Review by palatter — APPROVED** [review]

**Pull request merged**